crates/provider/src/kubernetes/scripts/zombie-wrapper.sh

```diff
@@ -20,7 +20,7 @@ fi
 echo "COMMANDS DEFINED"
 
 # add /cfg as first `looking dir` to allow to overrides commands.
-export PATH="{{REMOTE_DIR}}":$PATH
+export PATH="/cfg":$PATH
```

Comment:
We are setting `/cfg` as remote dir in the pod fs, let's just use the same here.